tutorials/tutorial_workflows_v2.md

```diff
@@ -0,0 +1,78 @@
+---
+permalink: /tutorial-workflows-v2
+title: Interacting with Workflows V2
+layout: default
+section_title: Guides and Tutorials
+---
+
+## Introduction
+
+(boilerplate explanation of the tool like V1's)
```

Comment:
I'll get a hand with writing this little blurb when reviewing this doc with product and our technical writer.